superset/assets/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -126,6 +133,11 @@ class SqlEditor extends React.PureComponent {
   }
   onSqlChanged(sql) {
     this.setState({ sql });
+    // Request server-side validation of the query text
+    if (this.canValidateQuery()) {
+      // NB. requestValidation is debounced
```

Comment:
What is NB?

Reply:
I never realized that was weird, but I guess it is a bit now that I think about it!

NB means "nota bene", latin for "note well" — I usually use it in comments where "NOTE:" would also work